README.md

```diff
@@ -117,7 +117,7 @@ The resulting parameters of the model estimation together with additional report
 As an example to get started you estimate the parameters of a quadrotor model with the reference log_files:
 
 ```
-make estimate-model model=quadrotor_model log=resources/quadrotor_model.ulg
+make estimate-model model=quadrotor_model log=resources/quadrotor_model.csv
```

Comment:
Why change the default test to a csv format?

Reply:
because the ulog contains the outlier. Having that case for getting started seems suboptimal to me. But the cleaner change would be to being able to short the ulog

Reply:
I would prefer the ulog since our default format is not a csv and it can be misleading if we need to convert a ulog to a csv